VH EFT samples added to nanoAODv5 for 2016 and 2017.

sharmaaash opened the pull request.

uajanssen commented:
I see you also modified other samples beyond the spin parity ones and I am wondering whether this PR is hence done one the latest sample list ?

sharmaaash commented:
> I see you also modified other samples beyond the spin parity ones and I am wondering whether this PR is hence done one the latest sample list ?

I am added spin/parity samples to nanoAODv5.I don't think I modified any other samples , Can you point me what sample I modified?

sharmaaash commented:
I think its not updated file in my config, I am creating another PR.
